pkg/instancetype/instancetype.go

```diff
@@ -527,6 +528,15 @@ func applyCpu(field *k8sfield.Path, instancetypeSpec *instancetypev1alpha2.Virtu
 		return Conflicts{field.Child("domain", "cpu")}
 	}
 
+	hasCPUGuest := false
```

Comment:
This function (`applyCpu()`) is used to populate field `vmi.spec.domain.CPU` with values from instancetype.
Instead of modifying this function, can you create a new one (for example `applyResources()`) that would populate field `vmi.spec.domain.resources` with values from `instancetype.spec.resources`?

Reply:
@opokornyy what do you think about this suggestion?